### PR TITLE
Tests for omniscidb-internal PR 5134 - fix multiple cursor argument support for UDTFs

### PR DESCRIPTION
**Pull request by pearu**
As in title. Requires https://github.com/omnisci/omniscidb-internal/pull/5134

### CODE REVIEW

File: rbc/tests/test_omnisci_column_arguments.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        define(o)
+        yield o
+
+
+def define(omnisci):
+
+    @omnisci('int32(Column<int64>, RowMultiplier, OutputColumn<int64>)')
+    def text_rbc_copy_c_rowmul(x, m, y):
+        for i in range(len(x)):
+            # TODO: implement is_null support
```
Comment (guilhermeleobas) [inline rbc/tests/test_omnisci_column_arguments.py:18]: This "TODO" refers to PR #230?

**Review by guilhermeleobas — APPROVED** [review]
LGTM. Just a small comment

File: rbc/tests/test_omnisci_column_arguments.py (hunk 1 above)
Comment (pearu) [reply rbc/tests/test_omnisci_column_arguments.py:18]: yes, right now the copy table functions are used for testing the rbc issue #187 but after the null support is in, these tests can be used for testing is_null support as well.

**Comment by pearu** [comment]
Merging in order to make `{table_name}null` table available to #230

**Pull request merged**